src/index.ts

```diff
@@ -874,10 +877,10 @@ export function HTPipe(...objs: any[]) {
               const rightOut = right.respond(leftOut.unsafeResponse);
               return {
                 unsafeResponse: rightOut.unsafeResponse,
-                status:
-                  rightOut.status === undefined
-                    ? leftOut.status
-                    : rightOut.status,
```

Comment:
this is needed because old implementations break the tests
if we don't return status it will be undefined but it still will be in return value of piped respond function
so it will break HTPipeTest function call because we can't pass status: undefined in pipeOut because of type mismatch but the js test will fail if we don't pass it because return value of piped respond function will have status: undefined and our pipe out don't so it will break the test

Reply:
good catch